src/service/action-impl.js

```diff
@@ -163,8 +163,11 @@ export class ActionService {
    */
   installActionHandler(target, handler) {
     const debugid = target.tagName + '#' + target.id;
-    user().assert(target.id && target.id.substring(0, 4) == 'amp-',
-        'AMP element is expected: %s', debugid);
+
+    // TODO: ASK @dima about this? Is this limited to AMP-elements for security
+    // reasons? If yes, shouldn't one check the tagname rather than the id?
+    //user().assert(target.id && target.id.substring(0, 4) == 'amp-',
+    //    'AMP element is expected: %s', debugid);
```

Comment:
Yeah. Probably would be ok to relax. The intention here is really to reduce a probability of error by installing this handler on an accidental element. But I just checked and it looks like `installActionHandler` is already a restricted API in presubmits. So we don't need to worry too much about misuse.

A slightly bigger problem is on the trigger side - where you also disabled `AMP` check. The issue is that if a publisher accidentally specifies the wrong target, they'll never know - a very easy mistake to make. So, I think, we still need some form of whitelist.